lib/muffin_man/product_fees/v0.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module MuffinMan
+  module ProductFees
```

Comment:
Also, can you add to the README that we now support this API?

Reply:
Done